.gitlab-ci.yml

```diff
@@ -0,0 +1,42 @@
+# GitLab CI in conjunction with GitLab Runner can use Docker Engine to test and build any application.
+# Docker, when used with GitLab CI, runs each job in a separate and isolated container using the predefined image that is set up in .gitlab-ci.yml.
+# In this case we use the latest python docker image to build and test this project.
+image: ska-registry.av.it.pt/ska-docker/tango-builder:latest
+
+variables:
+  DOCKER_DRIVER: overlay2
+
+services:
+  - docker:dind
+
+# cache is used to specify a list of files and directories which should be cached between jobs. You can only use paths that are within the project workspace.
+# If cache is defined outside the scope of jobs, it means it is set globally and all jobs will use that definition
+cache:
+  paths:
+
+# before_script is used to define the command that should be run before all jobs, including deploy jobs, but after the restoration of artifacts.
+# This can be an array or a multi-line string.
+before_script:
+  - docker login -u $DOCKER_REGISTRY_USER_LOGIN -p $CI_REGISTRY_PASS_LOGIN $CI_REGISTRY
+
+
+# The YAML file defines a set of jobs with constraints stating when they should be run.
+# You can specify an unlimited number of jobs which are defined as top-level elements with an arbitrary name and always have to contain at least the script clause.
+# In this case we have only the test job which produce an artifacts (it must be placed into a directory called "public")
+# It is also specified that only the master branch will be subject of this job.
+
+stages:
```

Comment:
I think it is better if we also publish the results as "pages" on SKA CI dashboard at this link:
https://ska-telescope.gitlab.io/ska_ci_dashboard/

In order to do that, we can take reference from DishMaster project and align the .gitlab.yml file accordingly. I am in the middle of applying this modification.

Reply:
Tried implementing the changes, but still, the dashboard is not reflecting the results. I will check with the System team on this issue.